cmd/collectors/restperf/restperf.go

```diff
@@ -619,7 +619,12 @@ func (r *RestPerf) PollData() (map[string]*matrix.Matrix, error) {
 						instance.SetLabel(display, f.value)
 						count++
 					} else {
-						r.Logger.Warn().Str("Instance key", instanceKey).Str("label", label).Msg("Missing label value")
+						// ignore physical_disk_id logging as in some of 9.12 versions, this field may be absent
+						if r.Prop.Query == "api/cluster/counter/tables/disk:constituent" && label == "physical_disk_id" {
+							r.Logger.Debug().Str("Instance key", instanceKey).Str("label", label).Msg("Missing label value")
```

Comment:
minor: better to use key without spaces and that matches var, `Str("instanceKey", instanceKey)`